src/renderer/src/constants/language.ts

```diff
@@ -1,10 +1,13 @@
+import { de, enUS, ptBR, zhCN, zhTW } from 'date-fns/locale'
```

Comment:
Instead of saving the objects in the available language, try it:
```typescript
import * as locales from 'date-fns/locale'

export const availableLanguages: TLanguage[] = [
  { label: 'Português (BR)', value: 'pt-BR', locale: 'ptBR' },
]

console.log(locales[availableLanguages[0].locale])
```